### ISSUE TITLE
`sparse-decoration` should not complain in tests

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/lints#411]
In tests it we are doing this to make an assertion:

```eo
+tests

# This is unit test.
[] > tests-compares-two-bools
  eq. > @
    true
    true
```

I suggest to not complain about the `sparse-decoration` if `+tests` meta is specified.

**Comment by h1alexbel** [comment]
@yegor256 @maxonfjvipon WDYT?

**Comment by yegor256** [comment]
@h1alexbel why not rewrite it:

```
+tests

# This is unit test.
eq. > tests-compares-two-bools
  true
  true
```

**Comment by h1alexbel** [comment]
@yegor256 aggree
